nominal/core/channel.py

```diff
@@ -114,6 +117,80 @@ def _from_conjure_logicalseries_api(
             _clients=clients,
         )
 
+    def get_decimated(
+        self,
+        start: str | datetime | IntegralNanosecondsUTC,
+        end: str | datetime | IntegralNanosecondsUTC,
+        *,
+        buckets: int | None = None,
+        resolution: int | None = None,
+    ) -> pd.DataFrame:
+        """Retrieve the channel data as a pandas.DataFrame, decimated to the given buckets or resolution.
+
+        Enter either the number of buckets or the resolution for the output.
+        Resolution in picoseconds for picosecond-granularity dataset, nanoseconds otherwise.
```

Comment:
Oof, this just silently switches behavior depending on the dataset precision? That's unfortunate if true

Reply:
It is stated like this in the docstring of SummarizeSeriesNode.resolution.